frontend/adapted_strength-app/src/pages/SignUp.jsx

```diff
@@ -0,0 +1,82 @@
+import logo from '../assets/logo.png';
+
+
+/*function validatePasswordMatch() {
+    var password = document.getElementById("password").value;
+    var password_conf = document.getElementById("password_conf").value;
+
+    if (password !== password_conf) {
+        return false;
+    } else {
+        return true;
+    }
+}
+*/
+
+
+function EmailField() {
+    return (<input type="email" placeholder="Email Address" id="email" name="email" required/>);
+}
+function PasswordField() {
+    return (<input type="text" placeholder="Password" id="password" name="password" pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,255}$" 
+    title="Password must contain at least one lowercase letter, one uppercase letter, one digit, and one special character. It should be between 8 and 255 characters long."
+    /*onchange="validatePasswordMatch()"*/ required/>);
+}
+function PasswordConfirmationField() {
+    return (<input type="text" placeholder="Re-Enter Password" id="password_conf" name="password_confirmation" /*onchange="validatePasswordMatch()"*/ required/>);
+}
+function SubmitButton() {
+    return (<button type="submit" className="border-slate-50 border-8 bg-black text-slate-200 rounded-full px-3 py-1 "  >Next</button>);
+}
+
+function HandleSubmit(event){
+    event.preventDefault();
+    //if (validatePasswordMatch()){
+        window.location.href="/sign-up-additional";
+        //store info
+        this.reset();
+    //}
```

Comment:
fix indentation and remove comments if not needed

Reply:
Mentioned in task: ADAPTEDS-89